common.yml

```diff
@@ -1690,6 +1766,19 @@ buildvariants:
     USE_SSL: "true"
   tasks: *ubuntu1804_arm64_tasks
 
+- name: ubuntu2004-arm64
+  display_name: ZAP ARM64 Ubuntu 20.04
+  run_on:
+    - ubuntu2004-arm64-test
+  stepback: false
+  batchtime: 10080 # weekly
```

Comment:
What's the rationale for these settings?

Reply:
Frankly, I add them here because they are present in the rest of the tests on the same hardware. I guess we just want to make sure the same test runs at most once/week in order to save the resource? 
@xdg Do you know the exact reason? I saw you added the original settings for the rest of the tests. Thanks!

Reply:
Yes, the ZAP platforms are elastic -- they are fixed static hosts, so we want to check them occasionally but leave them available for other projects to test against.

Reply:
SGTM. For some reason, I thought batchtime was "run at least this often, even if there was no new commit".